<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mytodowindow</class>
 <widget class="QWidget" name="mytodowindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我的待办</string>
   </property>
  </widget>
  <widget class="mylineedit" name="lineEdit_newtodo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+ 新建待办</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_todolists">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>1100</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>130</y>
     <width>420</width>
     <height>530</height>
    </rect>
   </property>
   <widget class="QListView" name="listView_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>420</width>
      <height>530</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_todoname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/time.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/remind.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/description.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>295</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/place.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/note.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="customerqlabel" name="label_dtime">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="customerqlabel" name="label_remindtime">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_desc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_place">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_note">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>高</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/image/high.png</normaloff>:/image/high.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/image/mid.png</normaloff>:/image/mid.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>低</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/image/low.png</normaloff>:/image/low.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>无优先级</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/image/null.png</normaloff>:/image/null.png</iconset>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>100</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>按到期时间排序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按创建时间排序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按优先级排序</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>显示已完成</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>760</y>
     <width>1100</width>
     <height>220</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>670</y>
     <width>420</width>
     <height>320</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_name_already">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
      <strikeout>true</strikeout>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/place.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/note.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/description.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/remind.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/time.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_level_already_icon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_level_already">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time_already">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_remind_already">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>420</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <zorder>listView</zorder>
   <zorder>label_name_already</zorder>
   <zorder>label_17</zorder>
   <zorder>label_16</zorder>
   <zorder>label_15</zorder>
   <zorder>label_14</zorder>
   <zorder>label_10</zorder>
   <zorder>label_level_already_icon</zorder>
   <zorder>label_level_already</zorder>
   <zorder>label_time_already</zorder>
   <zorder>label_remind_already</zorder>
   <zorder>textBrowser</zorder>
   <zorder>textBrowser_2</zorder>
   <zorder>textBrowser_3</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mylineedit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>customerqlabel</class>
   <extends>QLabel</extends>
   <header location="global">customerqlabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
